docker-compose.yml

```diff
@@ -76,7 +76,7 @@ services:
         logging:
             driver: none
     netaccess:
-        image: letsencrypt/boulder-tools-go${TRAVIS_GO_VERSION:-1.15.7}:2021-02-17
+        image: letsencrypt/boulder-tools:${BOULDER_TOOLS_TAG:-go1.15.7_a1df5bd3}
```

Comment:
Why a hash rather than a datestamp? I find the datestamp quite useful. We could do both - although also note that the git hash that shows up in these tags will almost certainly not match any git hash that gets committed to master, due to squashing.

Reply:
I don't have a strong preference. That is a good point about the hash not matching after a squash. I was trying to make it more auditable, like if we needed to push multiple versions in a day, but since the hash won't match it is probably better just to append the date. I'll push a change.